### PR TITLE
Return total on response size

### PR DESCRIPTION
**Pull request by vegetableman**
- Handles size for Informational status codes as well.

### CODE REVIEW

**Comment by vegetableman** [comment]
@shamasis have updated the size function.

File: lib/collection/response.js [hunk 1]
```diff
@@ -229,11 +229,13 @@ _.extend(Response.prototype, /** @lends Response.prototype */ {
         }
         // else size of body is added
         else {
-            sizeInfo.body = this.stream ? this.stream.byteLength : (this.body && this.body.length || 0);
+            sizeInfo.body = this.stream && this.stream.byteLength ? this.stream.byteLength :
```
Comment (shamasis) [inline lib/collection/response.js]: Was expecting the byte length based size computation here.

Comment (vegetableman) [inline lib/collection/response.js]: My bad. Missed these comments. Updated it.

File: lib/collection/header.js [hunk 2]
```diff
@@ -229,7 +229,7 @@ _.extend(Header, /** @lends Header */ {
         // status-line = HTTP-version SP status-code SP reason-phrase CRLF
         var raw = 'HTTP/X.X' + ' ' + statusCode + ' ' + reason + '\r\n';
         raw += this.unparse(headers, '\r\n');
-        raw += '\r\n\r\n';
+        raw += headers.count() ? '\r\n\r\n' : '\r\n';
```
Comment (shamasis) [inline lib/collection/header.js]: if headers were ever sent as array, then `.count` would raise errors.

Comment (vegetableman) [inline lib/collection/header.js]: Updated it.

**Comment by shamasis** [comment]
This

**Pull request closed without merging**